### PR TITLE
Fixed debian dependencies

### PR DESCRIPTION
**Pull request by mikhailnov**
* improved dependencies
* improved installation instructions
* added Ubuntu 18.10 cosmic to the PPA building script

Moved dracut-core, smartmontools to 'Suggests', inxi and numactl do not seem to have odd dependencies, so I left them in 'Recommends'

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -38,7 +38,6 @@ Content:
    - smartmontools
    - pciutils
    - usbutils
-   - edid-decode
```
Comment (lvc) [inline INSTALL]: It's required to collect EDIDs. Why moving to recommended?

File: debian/control [hunk 2]
```diff
@@ -8,8 +8,65 @@ Homepage: https://github.com/linuxhw/hw-probe
 
 Package: hw-probe
 Architecture: all
-Depends: perl-base (>=5), libdigest-sha-perl, curl, findutils, hwinfo, dmidecode, pciutils, usbutils, hdparm, smartmontools, edid-decode, util-linux, lsb-release, fdisk | util-linux, coreutils
```
Comment (lvc) [inline debian/control]: Need also smartmontools log in all probes.

**Comment by mikhailnov** [comment]
Fixed.
Smartmontools 6.6 with fixed dependencies has been built in the same PPA: https://launchpad.net/~mikhailnov/+archive/ubuntu/hw-probe/+packages

Also updated docs.

File: README.md [hunk 3]
```diff
@@ -28,39 +27,37 @@ You can make a probe with the help of a script, Docker image or Linux ISO (see b
 
 By creating probes you contribute to the "HDD/SSD Real-Life Reliability Test" study: https://github.com/linuxhw/SMART
 
-Usage
------
+## Usage
 
-    sudo hw-probe -all -upload -id DESC
+`sudo hw-probe -all -upload -id DESC`
```
Comment (linuxhw) [inline README.md]: This is not readable outside Github markdown viewer, so let's leave formatting for code as is.

File: README.md [hunk 4]
```diff
@@ -28,39 +27,37 @@ You can make a probe with the help of a script, Docker image or Linux ISO (see b
...
+	linuxhw/hw-probe -all -upload -id DESC
+```
 
 You may need to run `xhost +local:` before docker run to collect X11 info (xrandr, xinput, etc.).
 
 Docker hub repository: https://hub.docker.com/r/linuxhw/hw-probe/
 
-Live ISO
---------
+## Live ISO
```
Comment (linuxhw) [inline README.md]: Let's leave '----' for h2 and '####' for h3.

File: README.md [hunk 5]
```diff
@@ -69,25 +66,63 @@ The tool uploads SHA512 hash of MAC addresses and serial numbers to properly
...
+
+On **Ubuntu**-based Linux distributions (Ubuntu, Linux Mint, Elementary OS, etc.) you can install a PPA package:
+```
+sudo add-apt-repository universe
+sudo add-apt-repository ppa:mikhailnov/hw-probe
+sudo apt update
+sudo apt install hw-probe smartmontools
+```
+   
+If you want only a minimal set of dependencies to be installed, install as:
```
Comment (linuxhw) [inline README.md]: We need to install minimal set of deps by default.

File: README.md (hunk 3 above)
Comment (mikhailnov) [reply README.md]: Да вроде нормально читается в любом редакторе с минималньой подсветкой синтаксиса. Тут уже и конфликты автомержа подъехали, проще вам будет руками смержить, как нравится,а  не мне править.

**Comment by linuxhw** [comment]
Please squash all commits into one before creating pull requests. See https://help.github.com/articles/about-pull-request-merges/.

**Comment by lvc** [comment]
Cherry-picked/pushed effective commits from the request. Can't merge all due to the following commits:

* ce01ef8a2f498a7329449035a4ac6209f37c4819 (isn't based on master)
* 25753849259eabf6d88dea00122baae252c60c4f (wtf?)

Thank you.

**Comment by mikhailnov** [comment]
https://github.com/linuxhw/hw-probe/commit/25753849259eabf6d88dea00122baae252c60c4f is a merge-commit after I did:
```
git remote add upstream https://github.com/linuxhw/hw-probe.git
git fetch upstream
git merge upstream/master
```
to sync my fork mikhailnov/hw-probe with linuxhw/hw-probe. I don't know how to avoid such merge commits.

**Pull request merged**